.gitignore

```diff
@@ -13,3 +13,9 @@ notebooks/*
 examples/.ipynb_checkpoints/*
 .pytest_cache/*
 pytest/.pytest_cache/
+doc/tutorials/images
+doc/tutorials/index.rst
+doc/tutorials/run_*
+doc/tutorials/sg_execution_times.rst
```

Comment:
Maybe we should make a new folder (`tutorials_gbs`), and modify `conf.py` to place the built examples there.

That way it simplifies the gitignore, you just need to have
```
doc/tutorials_gbs/*
```

Reply:
:+1: makes sense to also not pollute `doc/tutorials/`!

Have done that now: https://github.com/XanaduAI/strawberryfields/pull/202/commits/a79c7a51181f1b879599e7a43f3c97cab6ccab9b. I guess in the long run it will be changed from `tutorials_gbs` to make it less specific to the apps layer, but this can be done with the upcoming PR to unify docs.

Reply:
Yep, agree! The PR to unify the SF tutorials and convert them into galleries can be where the folders are merged.